.github/workflows/copy-to-rhisv.yml

```diff
@@ -0,0 +1,39 @@
+name: Mirror images to RHISV Quay Org
+
+on:
+  workflow_call:
+    inputs:
+      sourceImageName:
+        required: true
+        type: string
+      sourceImageTag:
+        required: true
+        type: string
+    secrets:
+      sourceImageRegistry:
+        required: true
+
+env:
+  DEST_IMAGE_NAME: preflight-test
+
+jobs:
+  mirror-images-to-rhisv:
+    needs: []
+    name: Mirror Images
+    runs-on: ubuntu-latest
+    steps:
+    - name: Podman Login
+      uses: redhat-actions/podman-login@v1
+      with:
+        registry: ${{ secrets.RHISV_IMAGE_REGISTRY }}
```

Comment:
Probably best to pass these in as an input secret? Rather than refer to secrets directly?

Reply:
I think I prefer to see them as an input. Otherwise, you'd have to look in 2 places to know which secrets to set. As parameters, then you can see in the main workflow what needs to be set. Not a showstopper if you feel otherwise.

Reply:
Oh, and it makes this more reusable, should we need to add further locations.

Reply:
Fair enough, reconfigured this to use inputs almost entirely